.php_cs.dist

```diff
@@ -10,21 +20,16 @@ file that was distributed with this source code.
 HEADER;
 
 $finder = PhpCsFixer\Finder::create()
-    ->in(__DIR__)
-    ->exclude('tests/Fixtures/app/cache')
+    ->in([__DIR__.'/src', __DIR__.'/tests'])
```

Comment:
@bendavies I thought you've reverted this. We do want to include files in `bin` and `public` too, for example.

Can you explain the rationale for this change?